docs/reference/ontology-to-tools-compilation/supplementary.md

```
@@ -0,0 +1,980 @@
+---
+title: "Ontology-to-Tools Compilation — Supplementary Information (p.28-47)"
+source: "Zhou, X. et al. (2025). Ontology-to-tools compilation for executable semantic constraint enforcement in LLM agents. arXiv:2602.03439"
+notice: "Reference copy for internal analysis"
+---
+
+#### **7 Supplementary Information**
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)